scripts/backup

```diff
@@ -1,23 +1,57 @@
 #!/bin/bash
 
-# Exit on command errors and treat unset variables as an error
-set -eu
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+if [ ! -e _common.sh ]; then
+	# Get the _common.sh file if it's not in the current directory
+	cp ../settings/scripts/_common.sh ./_common.sh
+	chmod a+rx _common.sh
+fi
+source _common.sh
```

Comment:
I think @zamentur stated that this way was incorrect (I don't remember the precise reason...), and that we should simply `source ../settings/scripts/_common.sh`.

Reply:
According to the example, I should change it yes
https://github.com/YunoHost/example_ynh/blob/master/scripts/upgrade#L9

Reply:
Maybe having a YEP on the subject is possible? Adding it the `package_check` too ?
(I have no idea about the amount of work needed and if it is worth it to implement it)

Reply:
Found it @JimboJoe : https://github.com/YunoHost/yunohost/pull/489#issuecomment-396854686

@zamentur and others, is there any decision about this ? Shall the example_ynh be modified as well as all the apps ?

Reply:
Thanks for searching the reference!
My opinion is that we should update the example_ynh accordingly.

Reply:
What is the "correct way" of writing this? I am not sure

```
source ../settings/scripts/_common.sh
source _common.sh
source /usr/share/yunohost/helpers
```

or 

```
source _common.sh
source ../settings/scripts/_common.sh
source /usr/share/yunohost/helpers
```

or another way ?

Reply:
`source ../settings/scripts/_common.sh` is enough

Reply:
Just to be sure, why in this order?

And not the reverse like :

```
source /usr/share/yunohost/helpers
# "_common.sh" is loaded after official helpers for not breaking application
# if there is a change in official helpers and that application has not updated yet
source ../settings/scripts/_common.sh
```

Reply:
The reason of this order is exactly the opposite of that, we want to be sure that the official helpers will overwrite any old version of the helpers in _common.sh.
An update of an official helper should never break any apps.

Reply:
Well, I understand now. Thanks for this detailed explanation !

Reply:
This modification is only needed for the script `backup` ? Maybe `restore` too ?

Reply:
You can do the same thing for the restore script.